src/index.ts

```diff
@@ -7,15 +7,12 @@
 
 import * as ConfigStore from 'configstore';
 import {createHash} from 'crypto';
-import {GaxiosError, GaxiosOptions, GaxiosPromise} from 'gaxios';
+import {GaxiosError, GaxiosOptions, GaxiosPromise, GaxiosResponse} from 'gaxios';
 import {GoogleAuth, GoogleAuthOptions} from 'google-auth-library';
 import * as Pumpify from 'pumpify';
-import * as r from 'request';
-import {PassThrough} from 'stream';
+import {PassThrough, Transform} from 'stream';
 import * as streamEvents from 'stream-events';
 
-const request = r.defaults({json: true, pool: {maxSockets: Infinity}});
```

Comment:
So I have no idea what `pool: {maxSockets: Infinity}` actually does, or if it's important.  Is this something we need to translate to gaxios speak?

Reply:
`maxSockets` is the number of concurrent connections to the same host that are allowed. This was borrowed from our common code, which removes the limit that Node ships with by default. Node 6 and up set this to `Infinity` by default anyway, so I don't think we need this anymore.